dace/libraries/sparse/nodes/csrmm.py

```diff
@@ -524,13 +533,6 @@ def expansion(node, state, sdfg):
             nstate.add_edge(tasklet, '_conn_c', gc, None, dace.Memlet.from_array('_c_gpu', cdesc))
             nstate.add_nedge(gc, c, dace.Memlet.from_array('_c', cdesc))
 
-            if node.beta != 0.0:
```

Comment:
why was this removed?

Reply:
> why was this removed?

Because I added a pure-Expansion + GPUTransformSDFG for the case of beta != 0 before the outer if statement

I can split this into two commits?